libs/core/src/components/pds-avatar/pds-avatar.tsx

```diff
@@ -79,13 +82,13 @@ export class PdsAvatar {
       this.dropdown
         ?
         <button class="pds-avatar__button" type="button">
-          <div style={style}>
+          <div style={style} part="image">
```

Comment:
Totally up to you but since the avatar can contain an image or icon, may be another name could be used for the "wrapper" div.

Should we possibly add additional icon and image parts to those elements as well?

Just request another review if you do change the name or add additional parts, otherwise LGTM!